### DISCUSSION TITLE
Configuring security SLAs

### DISCUSSION THREAD

**Discussion by dduzgun-security** [community/community#43777] [Code Security]
### Problem encountered
I have no way of tracking progress on security vulnerability resolution over time to see if it respects my security resolution SLAs.

### Suggestion

In the configuration of the GitHub Organization 
- Adding a section for **security SLAs** where owners or security managers can set resolution delays for found vulnerabilities.
- Tracking the respect of the SLAs and resolutions over time (dashboard would look great)
- Being notified if the delay is breached

**Comment by kellyarwine** [comment]
Hi, @dduzgun-security.  Thank you for the feedback.  You'll be pleased to know this feature is on our roadmap. That being said, it's currently slated for a longer-term implementation (~1 year). Our development schedule can be a bit fluid when features are that far out. Once we're closer to implementing, you can check out [public roadmap](https://github.com/orgs/github/projects/4247/views/1?filterQuery=-status%3A%22Q4+2022+%E2%80%93+Oct-Dec%22+label%3A%22github+advanced+security%22++-label%3Ashipped+) as I'll post it there. Stay tuned!

**Reply by wildan3105** [reply]
Hi, any updates on this? Also wanted to ask if this data (security vulnerability statistics) is currently available via API? @kellyarwine 

Thank you.

**Reply by orosander** [reply]
@kellyarwine Any updates on this close to 3 years down the road? It would be a great feature for any security team!
